src/ctapipe/image/muon/ring_fitter.py

```diff
@@ -32,22 +51,44 @@ class MuonRingFitter(Component):
 
     fit_method = traits.CaselessStrEnum(
         list(FIT_METHOD_BY_NAME.keys()),
-        default_value=list(FIT_METHOD_BY_NAME.keys())[0],
+        default_value="kundu_chaudhuri_taubin",
     ).tag(config=True)
 
-    def __call__(self, fov_lon, fov_lat, img, mask):
-        """allows any fit to be called in form of
-        MuonRingFitter(fit_method = "name of the fit")
+    def __call__(self, geom, image, clean_mask):
+        """
+        Parameters
+        -----------
+        geom: CameraGeometry
+        Defines the pixel coordinates
```

Comment:
The descriptions need to be indented here, otherwise the docs won't render correctly. But otherwise, yes this looks nicer, and makes it clearer how to use it.  

Looking at it in more detail, you should  also mention in the description of the `geom` field that it must be in the `TelescopeFrame`. 

For correct NumPyDoc strings, you have to do it like this:

```python
Parameters
----------
parameter: type
    description indented by 4 spaces
    can go on multiple lines
```

so 
```suggestion
    Defines the pixel coordinates. 
    Must be in the `ctapipe.coordinates.TelescopeFrame`
```

Reply:
The do string should also start with a one-line summary what the method does, not directly with the parameters